lightningd/plugin.c

```diff
@@ -203,6 +236,32 @@ bool plugin_blacklisted(struct plugins *plugins, const char *name)
 	return false;
 }
 
+void plugin_important(struct plugins *plugins, const char *name)
+{
+	tal_arr_expand(&plugins->important,
+		       tal_strdup(plugins->important, name));
+}
+
+bool plugin_unimportant(struct plugins *plugins, const char *name)
+{
+	size_t l = tal_count(plugins->important);
```

Comment:
it'd be easier to read if `l` had a more descriptive name (`l` and `i` are also visually similar)